src/main/tv/codely/scala_http_api/module/course/domain/CourseId.scala

```diff
@@ -0,0 +1,9 @@
+package tv.codely.scala_http_api.module.course.domain
```

Comment:
Since this `CourseId` class is the one we share between some modules, it should go in the `shared` module. This way we can use it in order to represent the `Video#courseId` attribute type without coupling the `video` module to the `course` one 🙂